### PR TITLE
Added app description to readme config

### PR DESCRIPTION
**Pull request by Manpreet-Bhatti**

### CODE REVIEW

**Comment by Manpreet-Bhatti** [comment]
@sahandfarahani lmk if everything is fine on your end and I'll request review from ralph too :D

File: readme.config.js [hunk 1]
```diff
@@ -7,8 +7,12 @@
...
                         "WithName":true,
+                        "WithEmail":true,
                         "WithLocation":true,
+                        "WithTwitterUsername":true,
+                        "WithNumberOfRepos":true
                       },
                       "TableDesign":"CELL_DESIGN",
-    }
+    },
+    "AppDescription": "",
```
Comment (sahandfarahani) [inline readme.config.js:17]: You don't need to put initial value inside the config.js... if you want to change anything you can do it here...
just make sure to add AppDescription inside the table I made for descriptions for config options

File: src/App.tsx [hunk 2]
```diff
@@ -364,4 +371,21 @@ const App: React.FC = () => {
 
 export default App;
 
+const LightBGColor = styled.div`
```
Comment (sahandfarahani) [inline src/App.tsx:374]: I think you can delete LightBGColor and DarkBGColor  from here... I added them inside another component already

**Review by sahandfarahani — APPROVED** [review]
LGTM

File: readme.config.js (hunk 1 above)
Comment (Manpreet-Bhatti) [reply readme.config.js:17]: Which file would that table be in? @sahandfarahani

**Comment by Manpreet-Bhatti** [comment]
@sahandfarahani How about now?

**Comment by sahandfarahani** [comment]
> @sahandfarahani How about now?

LGTM

**Comment by Manpreet-Bhatti** [comment]
@ralph-dev just want to get your final approval before merging

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**